sophos_central_cloud/manifest.json

```diff
@@ -10,7 +10,18 @@
     "changelog": "CHANGELOG.md",
     "description": "Gain insights into Sophos Central Cloud logs.",
```

Comment:
"Gain insights into Sophos Central Cloud alert and event logs."

Reply:
We have updated description.